docs/software/advanced_networking.md

```diff
@@ -145,6 +145,59 @@ The WiFi HotSpot package turns your device into a wireless hotspot/access point.
 
     Additionally, DHCP leases can be monitored via the file `/var/lib/dhcp/dhcpd.leases`. 
 
+=== "Combine with AdGuard Home resp. Pi-Hole"
+
+    The WiFi HotSpot can be combined with AdGuard Home resp. Pi-Hole. To do this, the DHCP server needs to know the IP address of the Ad-Blocker system and announces it as the DNS server for the WiFi area.
+
+    The corresponding file is `/etc/dhcp/dhcpd.conf`, the entry `option domain-name-servers` has to be set to your Ad-Blocker IP address.  
+    In the following there are three examples given starting from the following base `dhcpd.conf` content:
+
+    ```
+    authoritative;
+    #default-lease-time 43200;
+    #max-lease-time 86400;
+
+    subnet 192.168.42.0 netmask 255.255.255.0 {
+        range 192.168.42.10 192.168.42.250;
+        option broadcast-address 192.168.42.255;
+        option routers 192.168.42.1;
+        option domain-name "local";
+        option domain-name-servers 9.9.9.9, 149.112.112.112;
+    }
+    ```
+
+    1. AdGuard Home runs on the same system as the WiFi HotSpot runs. Then the "subnet" section contents has to be changed to
+        ```
+        subnet 192.168.42.0 netmask 255.255.255.0 {
+            range 192.168.42.10 192.168.42.250;
+            option broadcast-address 192.168.42.255;
+            option routers 192.168.42.1;
+            option domain-name "local";
+            option domain-name-servers 192.168.42.1;
+        }
+        ```
+
+    1. Pi-Hole runs in the subnet the LAN connection belongs to. Depending on the WAN subnet (e.g. 192.168.178.0/24) the "subnet" section contents might be changed to
```

Comment:
```suggestion
    1. Pi-Hole runs in the subnet the LAN connection belongs to. Depending on the LAN subnet (e.g. 192.168.178.0/24) the "subnet" section contents might be changed to
```
I know what you mean, but "WAN" sounds more like the outside network behind the router. Probably we should keep naming it "LAN" and differentiate it from "WiFi network"?

Also, probably we should probably make clear that that this way Pi-hole answers requests on both networks (LAN and WiFi), which is only possible for the WiFi network thanks to IP forwarding rules for shared Internet access for WiFi clients. Otherwise access from WiFi clients to LAN-side hosts wouldn't be possible. In Pi-hole, one must still explicitly allow queries from all origins, as it sees them coming from a different subnet, right?